From fe64d2622d6fd880f35da93382e1de5077b681f3 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Wed, 29 Apr 2020 10:37:08 +0000
Subject: [PATCH 24/24] WIP: ARM: dts: exynos5422: HC1/XU3/XU4 model name is
 ODROID not Odroid

Cosmetic change to model and audio card name for HC1/XU3/XU4 to
put ODROID in capitals (as per Hardkernel branding). Also fixup
some unneeded backslashes and wrap lines per kernel standards.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm/boot/dts/exynos5422-odroidhc1.dts        | 4 ++--
 arch/arm/boot/dts/exynos5422-odroidxu3-audio.dtsi | 2 +-
 arch/arm/boot/dts/exynos5422-odroidxu3-lite.dts   | 5 +++--
 arch/arm/boot/dts/exynos5422-odroidxu3.dts        | 5 +++--
 arch/arm/boot/dts/exynos5422-odroidxu4.dts        | 6 +++---
 5 files changed, 12 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/exynos5422-odroidhc1.dts b/arch/arm/boot/dts/exynos5422-odroidhc1.dts
index 812659260278..b90da73510cf 100644
--- a/arch/arm/boot/dts/exynos5422-odroidhc1.dts
+++ b/arch/arm/boot/dts/exynos5422-odroidhc1.dts
@@ -11,8 +11,8 @@
 #include "exynos5422-odroid-core.dtsi"
 
 / {
-	model = "Hardkernel Odroid HC1";
-	compatible = "hardkernel,odroid-hc1", "samsung,exynos5800", \
+	model = "Hardkernel ODROID HC1";
+	compatible = "hardkernel,odroid-hc1", "samsung,exynos5800",
 		     "samsung,exynos5";
 
 	pwmleds {
diff --git a/arch/arm/boot/dts/exynos5422-odroidxu3-audio.dtsi b/arch/arm/boot/dts/exynos5422-odroidxu3-audio.dtsi
index c3c2d85267da..26961dcea010 100644
--- a/arch/arm/boot/dts/exynos5422-odroidxu3-audio.dtsi
+++ b/arch/arm/boot/dts/exynos5422-odroidxu3-audio.dtsi
@@ -13,7 +13,7 @@
 / {
 	sound: sound {
 		compatible = "samsung,odroid-xu3-audio";
-		model = "Odroid-XU3";
+		model = "ODROID-XU3";
 
 		samsung,audio-widgets =
 			"Headphone", "Headphone Jack",
diff --git a/arch/arm/boot/dts/exynos5422-odroidxu3-lite.dts b/arch/arm/boot/dts/exynos5422-odroidxu3-lite.dts
index 98feecad5489..d0084033199a 100644
--- a/arch/arm/boot/dts/exynos5422-odroidxu3-lite.dts
+++ b/arch/arm/boot/dts/exynos5422-odroidxu3-lite.dts
@@ -14,8 +14,9 @@
 #include "exynos54xx-odroidxu-leds.dtsi"
 
 / {
-	model = "Hardkernel Odroid XU3 Lite";
-	compatible = "hardkernel,odroid-xu3-lite", "samsung,exynos5800", "samsung,exynos5";
+	model = "Hardkernel ODROID XU3 Lite";
+	compatible = "hardkernel,odroid-xu3-lite", "samsung,exynos5800",
+		"samsung,exynos5";
 };
 
 &arm_a7_pmu {
diff --git a/arch/arm/boot/dts/exynos5422-odroidxu3.dts b/arch/arm/boot/dts/exynos5422-odroidxu3.dts
index db0bc17a667b..3ff7ec514e20 100644
--- a/arch/arm/boot/dts/exynos5422-odroidxu3.dts
+++ b/arch/arm/boot/dts/exynos5422-odroidxu3.dts
@@ -13,8 +13,9 @@
 #include "exynos54xx-odroidxu-leds.dtsi"
 
 / {
-	model = "Hardkernel Odroid XU3";
-	compatible = "hardkernel,odroid-xu3", "samsung,exynos5800", "samsung,exynos5";
+	model = "Hardkernel ODROID XU3";
+	compatible = "hardkernel,odroid-xu3", "samsung,exynos5800",
+		"samsung,exynos5";
 };
 
 &i2c_0 {
diff --git a/arch/arm/boot/dts/exynos5422-odroidxu4.dts b/arch/arm/boot/dts/exynos5422-odroidxu4.dts
index 892d389d6d09..d2061b244537 100644
--- a/arch/arm/boot/dts/exynos5422-odroidxu4.dts
+++ b/arch/arm/boot/dts/exynos5422-odroidxu4.dts
@@ -13,8 +13,8 @@
 #include "exynos5422-odroidxu3-common.dtsi"
 
 / {
-	model = "Hardkernel Odroid XU4";
-	compatible = "hardkernel,odroid-xu4", "samsung,exynos5800", \
+	model = "Hardkernel ODROID XU4";
+	compatible = "hardkernel,odroid-xu4", "samsung,exynos5800",
 		     "samsung,exynos5";
 
 	pwmleds {
@@ -31,7 +31,7 @@
 
 	sound: sound {
 		compatible = "samsung,odroid-xu3-audio";
-		model = "Odroid-XU4";
+		model = "ODROID-XU4";
 
 		samsung,audio-routing = "I2S Playback", "Mixer DAI TX";
 
-- 
2.20.1

